joomla/joomla-cms
# [4.0] Cassiopeia dropdown accessible

Draft for https://github.com/joomla/joomla-cms/pull/29413.

### Summary of Changes

1. I added https://github.com/NickDJM/accessible-menu so that the **top menu** can used as drop down and a11y is fine.
2. I also made it possible, that `templates/cassiopeia/js/template.js` is minified via `build and npm`. 

There is no styling. (I changed the font color so that you can read the texts.)

### Testing Instructions

1. Apply this branch, delete the file `templates/cassiopeia/js/template.min.js` and run `npm ci`

2. Install `Testing Sample Data `
![Control Panel testq Administration](https://user-images.githubusercontent.com/9974686/86463971-d94bf300-bd2e-11ea-95e4-90fbc0dae5be.png)

3. Add some Menu Items in different levels to the top menu. If a menu item is drop down you should set it as system link
![Menus Edit Item testq Administration](https://user-images.githubusercontent.com/9974686/86463939-c46f5f80-bd2e-11ea-8a85-86cdc916071a.png)




### Actual result BEFORE applying this Pull Request
Drop Down is ignored


### Expected result AFTER applying this Pull Request
![Sample Sites](https://user-images.githubusercontent.com/9974686/86463628-2a0f1c00-bd2e-11ea-86fe-10efafe16ed5.png)
![Getting Started](https://user-images.githubusercontent.com/9974686/86463631-2aa7b280-bd2e-11ea-9bd0-87e78e83d2c7.png)



### Documentation Changes Required
Not sure
